service/ibmmq-plugin/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.solace.maas.plugin.ibmmq</groupId>
+    <artifactId>ibmmq-plugin</artifactId>
+    <version>1.6.3-SNAPSHOT</version>
+    <name>Solace Event Management Agent - IBM MQ Plugin</name>
+    <description>Solace Event Management Agent - IBM MQ Plugin</description>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <solace.maas.ema.version>1.6.3-SNAPSHOT</solace.maas.ema.version>
+        <spring.boot.version>3.1.8</spring.boot.version>
```

Comment:
I believe we use `spring.boot.version` 3.1.9 in other places, was there a specific reason you didn't go with that version?